src/valgrind_ctime_test.c

```diff
@@ -181,6 +185,46 @@ int main(void) {
     }
 #endif
 
+#ifdef ENABLE_MODULE_ECDSA_ADAPTOR
+    {
+        unsigned char adaptor_sig[162];
+        unsigned char deckey[32];
+        unsigned char expected_deckey[32];
+        secp256k1_pubkey enckey;
+
+        for (i = 0; i < 32; i++) {
+            deckey[i] = i + 2;
+        }
+
+        ret = secp256k1_ec_pubkey_create(ctx, &enckey, deckey);
+        VALGRIND_MAKE_MEM_DEFINED(&enckey, sizeof(secp256k1_pubkey));
+        VALGRIND_MAKE_MEM_DEFINED(&ret, sizeof(ret));
```

Comment:
Both of these should be unnecessary. The first one is not, in fact due to a bug that [we're fixing upstream](https://github.com/bitcoin-core/secp256k1/pull/894). If you don't want to wait for the upstream PR to land in secp-zkp, you can just cherry-pick my commit there.